resources/ory/charts/oathkeeper/templates/ingress-api.yaml

```diff
@@ -32,8 +32,8 @@ spec:
         {{- range .paths }}
           - path: {{ . }}
             backend:
-              serviceName: {{ $fullName }}-api
-              servicePort: http
+              serviceName: {{ $fullName }}-api # TODO: change or leave?
```

Comment:
? seems to me you have some doubts

Reply:
ORY chart uses `Ingress` resource with version `extensions/v1beta1`. We use `networking.k8s.io/v1` on our side. As a result, in our chart the marked key seems to be no longer needed (IDE prompts that this key is not expected). Should we then consider removing it or not?